styles/partials/blog-preview.scss

```diff
@@ -15,6 +15,7 @@
   border: var(--border, var(--default-border));
   display: flex;
   position: relative;
+  background-color: var(--bg);
```

Comment:
This is to remove the transparency from the original card for more solid view transitions